src/fava/json_api.py

```diff
@@ -114,12 +114,15 @@ def payee_accounts() -> List[str]:
 def query_result():
     """Render a query result to HTML."""
     query = request.args.get("query_string", "")
-    table = get_template_attribute("_query_table.html", "querytable")
```

Comment:
[Global variables](https://github.com/beancount/fava/blob/a8c4f03c1a3b2a5c452858b71b84887782e91602/src/fava/application.py#L201) are not set when using `flask.get_template_attribute`, which causes a test failure in `test_api_query_result` , so I changed a bit here.



https://stackoverflow.com/questions/19501628/get-template-attribute-how-to-inject-template-globals/65758207#65758207

Reply:
As mentioned elsewhere, I'd prefer it to pass `ledger` as a macro argument, then we could just use Jinja2's cached template and keep the code here working like before.

Reply:
Sorry about the slow progress. The PR is update now.